nixos/modules/programs/cfs-zen-tweaks.nix

```diff
@@ -0,0 +1,28 @@
+# CFS Zen Tweaks
+
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
Not using with lib; over the entire file is encouraged.

Reply:
Why? Does it make a difference in evaluation time?